openshift/patches/0002-default-build.patch

```diff
@@ -1,13 +1,13 @@
-diff --git a/cmd/build.go b/cmd/build.go
-index 3fc433bd..9845ed7e 100644
---- a/cmd/build.go
-+++ b/cmd/build.go
-@@ -50,7 +50,7 @@ and the image name is stored in the configuration file.
- 		PreRunE:    bindEnv("image", "path", "builder", "registry", "confirm", "push", "builder-image", "platform"),
- 	}
+diff --git a/function_builder.go b/function_builder.go
+index 38c6dc98..b6b34347 100644
+--- a/function_builder.go
++++ b/function_builder.go
+@@ -7,7 +7,7 @@ import (
+ const (
+ 	BuilderPack    = "pack"
+ 	BuilderS2i     = "s2i"
+-	DefaultBuilder = BuilderPack
++	DefaultBuilder = BuilderS2i
+ )
  
--	cmd.Flags().StringP("builder", "b", "pack", "build strategy to use when creating the underlying image. Currently supported build strategies are 'pack' and 's2i'.")
-+	cmd.Flags().StringP("builder", "b", "s2i", "build strategy to use when creating the underlying image. Currently supported build strategies are 'pack' and 's2i'.")
```

Comment:
https://github.com/knative-sandbox/kn-plugin-func/blob/df022f5c93911687e4356f4c58ccbb5d10b7a895/cmd/build.go#L52

This line has been changed upstream, so it shouldn't be needed to change the default value in here, once the `DefaultBuilder` is being patched.

Reply:
Well we are not changing it, that's what this PR does, am I misunderstanding something?

Reply:
Just look at the file directly https://github.com/openshift-knative/kn-plugin-func/blob/37b25e37c3e458c9c7b39341533ac3dca5738640/openshift/patches/0002-default-build.patch

Reply:
You may be confused by the this is diff of a diff.

Reply:
It confuses me too many times.